configman/config_manager.py

```diff
@@ -461,7 +461,10 @@ def write_conf(self, config_file_type, opener, skip_keys=None):
                 an_option = option_defs[a_key]
                 if ((not a_key.startswith('admin'))
                     and isinstance(an_option, Option)
-                    and an_option.secret
+                    and (
+                        an_option.secret
+                        or 'password' in an_option.name.lower()
+                    )
```

Comment:
You don't have to compromise just to satisfy me. I think "db_password" is possible. Maybe more possible and realistic than `password_hint` or `min_password_length`. We want to really do our best to prevent annoying and time consuming "leaks" however, I just think if we do my if statement idea we'd be able to reverse on it if we really need to. My idea depends on the `an_option.secret` becoming `None` by default if not explicitly set.